sycl/doc/extensions/experimental/sycl_ext_matrix/sycl_ext_oneapi_matrix.asciidoc

```diff
@@ -467,6 +467,9 @@ q.parallel_for(nd_range<2>(G, L), [=](nd_item<2> it) {
   }
 });
 ```
+In the case of `ext_intel_packed` and `col_major` matrix memory
+layout, `Rows` and `Cols` represent the logical shape of the matrix
+before VNNI or transpose transformation.
```

Comment:
Since this is the "oneapi" specification, it would be better not to reference things in the Intel-specific extension.  Could we say something like this instead:

> The `Rows` and `Cols` template parameters represent the logical shape of the matrix, which is not necessarily the way the matrix is layed out in memory.  Thus, these template parameters have the same meaning in `joint_matrix_prefetch` as they do for `joint_matrix_load`.